### DISCUSSION TITLE
音频数字水印

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#6] [Announcements]
# 音频数字水印

一些数字音乐商店采用水印来打击资源泄露，然而，一些拙劣的水印算法会造成可闻的听感损失。2020-08-22 是本文初稿完成的日期。

https://zexwoo.blog/posts/knowledge/audio-watermark/

**Comment by github-24** [comment]
大佬，苹果音乐的音源有水印吗

**Reply by ZexWoo** [reply]
据我所知没有。

**Reply by github-24** [reply]
大佬，Q音有水印吗？感觉音源比网易云好

**Reply by ZexWoo** [reply]
我个人认为 QQ 比网易靠谱一些，但是总的来说国内音乐平台都比较烂，听个乐差不多了，别较真。

**Reply by github-24** [reply]
据说抓取的资源是不带水印的，玄学

**Comment by jxspaozjy** [comment]
也就是mora源没有水印是吗

**Reply by ZexWoo** [reply]
据我所知没有。Mora 的问题是，提供的 FLAC 文件是无压缩文件，你下载以后可以自己再开 Level 8 压缩一下，可以节约大概 40% 的空间。

**Comment by dumbbird** [comment]
今天无意间找到一个去年下的一个FLAC音频文件，来源应该是deezer，版权来自环球。
并非hi-res。

恰好有一个原版CD抓轨的档案，因此两者可以进行对比。
结果比对一番之后，发现确实也存在水印。水印长这个样子。
https://www.hdcdape.com/data/attachment/forum/202407/28/234731dfmnmz3jb138epzb.jpg

确实很乐色啊环球！！！

不过我最近开始用TIDAL了，想研究下TIDAL（非MQA）的音乐质量，听说TIDAL如果不是MQA的话音质不错且无水印，不知真假。
